CatProducer/prod/crabConfig.py

```diff
@@ -19,7 +19,9 @@
 # Where the output files will be transmitted to
 #config.Site.storageSite = 'T2_KR_KNU'
 #crab checkwrite --site=T3_KR_KISTI --lfn=/store/group/CAT/
-config.Site.storageSite = 'T3_KR_KISTI'
-#config.Site.storageSite = 'T3_KR_UOS'
-config.Data.outLFNDirBase = '/store/group/CAT/' 
+#config.Site.storageSite = 'T3_KR_KISTI'
+config.Site.storageSite = 'T3_KR_UOS'
+#config.Data.outLFNDirBase = '/store/group/CAT/' 
+config.Data.outLFNDirBase = '/store/user/jipark/'
```

Comment:
I think we (still) have to send output to KISTI by default

Reply:
Back to KISTI. Also I received a mail from Ryu that KISIT crab server now works okay.